projects/elfutils/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -eu
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build elf tooling targets
+autoreconf -i -f
+./configure --enable-maintainer-mode --disable-libdebuginfod --enable-libdebuginfod=dummy --disable-debuginfod --disable-libdebuginfod
```

Comment:
I wonder how it can be built with ASan in its current form? I had to change a couple of files with sed to get libelf to compile with ASan: https://github.com/google/oss-fuzz/blob/6fb4ca2c8342077b88f33984d1e05de26e5a3b02/projects/libbpf/build.sh#L74-L79

Or is https://github.com/izzeem/elfutils somehow different from the main repository hosted at git://sourceware.org/git/elfutils.git?

Reply:
These are the changes I made from the branch to support it. I just keep a separate branch so I can always pull in the upstream.

Reply:
https://github.com/izzeem/elfutils/pull/1